src/draw_box.c

```diff
@@ -474,8 +474,8 @@ s32 draw_box(s32 flags, WindowStyle windowStyle, s32 posX, s32 posY, s32 posZ, s
                 guMtxCatF(mtx2, rotScaleMtx, mtx1);
             }
             if (outMtx) {
-                for (i = 0; i < 4; i++) {
-                    for (j = 0; j < 4; j++) {
+                for (i = 0; i < COUNT_ROWS(mtx1); i++) {
```

Comment:
I think these would be better as ROW_COUNT & COL_COUNT , but also not so sure how necessary they are. it's not like you'd want to change these values

Reply:
I disagree